src/components/SearchResults/ExactMatch/index.js

```diff
@@ -39,13 +33,9 @@ class ExactMatchWrapper extends PureComponent /*:: <EMWProps> */ {
   render() {
     const { to, children } = this.props;
     return (
-      <div className={f('callout', 'warning', 'margin-bottom-medium')}>
-        <span className={f('icon', 'icon-common')} data-icon="&#xf35a;">
-          {' '}
-          Found an exact match:
-        </span>{' '}
-        <Link to={to}>{children}</Link>
-      </div>
+      <Callout type="warning" customIcon="icon-arrow-alt-circle-right">
```

Comment:
```suggestion
      <Callout type="warning" icon="icon-arrow-alt-circle-right">
```

if accepting [this suggestion](https://github.com/ProteinsWebTeam/interpro7-client/pull/567#discussion_r1471279716).